src/marqo/s2_inference/s2_inference.py

```diff
@@ -482,7 +482,7 @@ def _check_memory_threshold_for_model(device: str, model_size: Union[float, int]
         raise ModelCacheManagementError(
             f"You are trying to load a model with size = `{model_size}` into device = `{device}`, which is larger than the device threshold = `{threshold}`. "
             f"Marqo CANNOT find enough space for the model. Please change the threshold by adjusting the environment variables.\n"
-            f"You can find more detailed information at `https://docs.marqo.ai/0.0.21/Advanced-Usage/configuration/`.")
+            f"You can find more detailed information at `https://docs.marqo.ai/latest/other-resources/guides/advanced-usage/configuration/`.")
```

Comment:
Let's use `marqo_docs.configuring_marqo()` to reference the doc. Also, could you provide the environment variable name in the error message as well?

Reply:
This is just a temporary measure since Li indicated that he's adding `marqo_docs.configuring_marqo()` into his PR, but I will add the variable name!